### ISSUE TITLE
Umbrella Issue to note potential accessibility improvements

### ISSUE THREAD

**Issue by windymilla** [DistributedProofreaders/guiguts-py#1824]
1. "Decorative" `<hr>` elements, e.g. for chapter break, or surrounding an illo, rather than a true thought break that was probably part of the original text: add `aria-hidden="true"` to the `<hr>` element. See [Winnie-the-Pooh](https://gutenberg.org/cache/epub/67098/pg67098-images.html) for example.
2. Possibly put `<main>` (or `<main role="main">`) at start & `</main>` at end of book during HTML autogeneration. PG header footer will end up outside that when WWed. PPer can adjust position, e.g. to exclude adverts, for some books.
3. Consider putting `<nav>` tags around autogenerated ToC

**Comment by windymilla** [comment]
Notes for documentation:
1. `section` elements should be used for front matter, end/back matter - to be manually added by PPer
2. GG adds `nav` tags round auto-ToC. If PPer isn't using it, they should add them round their ToC
3. GG adds `aria-hidden="true"` to `hr` elements it adds between chapters. If user inserts additional decorative `hr` elements, they should include the same element. This is not needed for `hr` which represent a true thought break, e.g. `* * *` in text to indicate a thought break, that gets converted to `hr`
4. Accessibility stuff should be consolidated ( [HTML Best Practices Document](https://www.pgdp.net/wiki/DP_Official_Documentation:PP_and_PPV/DP_HTML_Best_Practices/Case_Studies/Inline_Formatting#presentational-vs-semantic), https://www.pgdp.net/wiki/Accessibility_Recipes/Emphasis, https://www.pgdp.net/wiki/Accessible_HTML_eBooks, etc). Then the PP FAQ should have a link (or links) to it.
5. WAVE report button (or instructions for doing it manually) should be documented.

**Comment by windymilla** [comment]
Tasks all complete. Documentation added to https://www.pgdp.net/wiki/Accessible_HTML_eBooks
